pkgs/os-specific/linux/nixos-rebuild/nixos-rebuild.sh

```diff
@@ -1,4 +1,5 @@
 #! @runtimeShell@
+# shellcheck shell=bash
```

Comment:
I am not sure if runtimeShell can be anything besides bash but setting it so sh would already be broken so I think it is fine setting this to bash right now.